rpc/metrics.go

```diff
@@ -110,13 +111,22 @@ var (
 		prometheus.HistogramOpts{
 			Namespace: "hmy",
 			Subsystem: "rpc",
-			Name:      "rpc_request_delay",
+			Name:      "request_delay_histogram",
 			Help:      "delay in seconds to do rpc requests",
-			// buckets: 20ms, 40ms, 80ms, 160ms, 320ms, 640ms, 1280ms, +INF
-			Buckets: prometheus.ExponentialBuckets(0.02, 2, 8),
+			// buckets: 20ms, 40ms, 80ms, 160ms, 320ms, 640ms, 1280ms, 2560ms, 5120ms, +INF
```

Comment:
I suggest use smaller number of buckets to keep track of the latency. Starting from 50ms, using 8 buckets

Reply:
Changed